assets/blocks/quiz/quiz-store.js

```diff
@@ -129,4 +146,20 @@ registerStructureStore( {
 	clearError() {
 		dispatch( 'core/notices' ).removeNotice( 'quiz-structure-save-error' );
 	},
+
+	/**
+	 * Remove derived elements from quiz response.
+	 *
+	 * @param {Object} structure The quiz response.
+	 *
+	 * @return {Object} The modified response.
+	 */
+	setServerStructure( structure ) {
+		return {
+			...structure,
+			questions: structure.questions.map( ( question ) =>
+				omit( question, [ 'categories', 'shared', 'student_help' ] )
```

Comment:
Good to know this array is here. I wonder about putting somewhere more obvious, like possibly a constant of read-only attributes? Not sure. I remember there was a reason why `student_help` is here, but I don't remember what it was. We'll also add the `editable` flag once we add it the the REST API.

Reply:
Good idea! Moved them to a constant in 13bcec6f37303556763cb53961050a384b65666c. 

Now that I think about it, it might be more clear if we expose read only attributes as an argument to `registerStructureStore` instead of exposing `setServerStructure`. But maybe exposing the method is more expandable?

`student_help` is an attribute of file upload which we intend to integrate in the question description. Maybe a better handling of it can be implemented when we work on the migration.